test/api.js

```diff
@@ -314,9 +314,14 @@ test('test file in node_modules is ignored', function (t) {
 test('Node.js-style --require CLI argument', function (t) {
 	t.plan(1);
 
+	var requirePath = path.relative(process.cwd(), path.join(__dirname, 'fixture', 'install-global.js'));
+	// `path` will use \\ to seperate paths on windows, but require doesn't.
+	requirePath = requirePath.split(path.sep).join('/');
```

Comment:
You can use `\\` in `require` and `--require`, but it only works on Windows. So it won't be cross-platform. We could normalize internally, but do we really want to implement it differently than `$ node --require`?